app/constants/Constants.scala

```diff
@@ -20,6 +20,7 @@ import scala.collection.immutable.ListMap
 /**
  * Created by xavierzanatta on 3/25/15.
  */
+//scalastyle:off magic.number
```

Comment:
See previous comment

Reply:
I've switched it back on at the end of the object